docs/csharp/tutorials/console-teleprompter.md

```diff
@@ -310,12 +309,12 @@ private static async Task GetInput()
 
 This creates a lambda expression to represent an `Action` that reads a key
```

Comment:
Here the link would be helpful -- @System.Action. Also, perhaps "Action delegate"?